up-l1/README.adoc

```diff
@@ -122,9 +122,21 @@ On the other hand, unsuccessful completion of this method does not necessarily m
 
 NOTE: The above strategy for retrying failed attempts to send a message results in https://www.cloudcomputingpatterns.org/at_least_once_delivery/[at-least-once delivery]. Recipient(s) of these messages should therefore be https://www.cloudcomputingpatterns.org/idempotent_processor/[Idempotent Processors].
 
-UTransport implementations
```

Comment:
FMPOV this line needs to remain in the document. Otherwise, the following list makes no sense ...

Reply:
I'll add the ":" then to the end

Reply:
I don't think I understand ...

Reply:
I added it back